tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "target": "es5",
+        "target": "es2015",
```

Comment:
What's the reason behind changing this?

Reply:
Not using ES2015 yields this error:
```
src/components/RightPane/CoursePane/SearchForm/AdvancedSearch.tsx:112:20 - error TS2802: Type 'IterableIterator<number>' can only be iterated through when using the '--downlevelIteration' flag or with a '--target' of 'es2015' or higher.

112             ...[...Array(10).keys()].map((v) => v + 2 + ':00am'),
                       ~~~~~~~~~~~~~~~~
```

If we need to stick with ES5 (e.g. for browser compatibility reasons) then we could probably include the `downlevelIteration` flag instead, but either way the tsconfig needs to be changed.